packages/mantine/src/theme/Theme.tsx

```diff
@@ -276,12 +276,12 @@ export const plasmaTheme: MantineThemeOverride = {
                 },
             },
         },
-        Segmented: {
-            styles: {
-                control: {
-                    zIndex: 'unset',
```

Comment:
you sure about removing this one ?

Reply:
`Segmented` doesn't exist, so it had no effect anyway.